examples/2d/frustum_sprites.rs

```diff
@@ -0,0 +1,83 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+};
+
+const CAMERA_SPEED: f32 = 1000.0;
+
+pub struct CameraTimer(Timer);
+pub struct Direction(Vec3);
```

Comment:
I wonder if a more circular camera motion is more helpful - that way the sprites are always on screen. In that case, you wouldn't need a timer since you would just move around the circle by `SPEED * time.delta_seconds()` radians.

Reply:
Ok, so looking at your video it looks like if this was performing properly then sprites would always be on the screen - the sprites being offscreen is just a quirt of time.deltra_time being 10 seconds (debug mode 😓 )